src/main/java/codeu/controller/LoginServlet.java

```diff
@@ -16,6 +16,8 @@
 
 import codeu.model.data.User;
 import codeu.model.store.basic.UserStore;
+import org.mindrot.jbcrypt.BCrypt;
```

Comment:
nit: Prefer keeping imports alphabetized. google-java-format *should* do this for you.